README.md

```diff
@@ -141,6 +142,15 @@ The following methods are exposed on an instantiated client
 | subquery | `(targetPath, sourceTree, sourcePath, mapSubqueryValues?) => {}` | Sets up a subquery, using the types passed in to the client and assuming this tree instance is the target tree. For more info, see the [subquery](#Subquery) section below. |
+| replace | `async (path, node) -> await searchCompleted` | Replaces a the node at the given path with the new node. You can await this for when updates settle and relevant searches are completed. |
+| indent | `async (path, newNode) -> await searchCompleted` | Indents the node at the provided path by either `indentInPlace` or `indentReplace` depending on whether the target is the root node or not. You can await this for when updates settle and relevant searches are completed. |
+| indentReplace | `async (path, newNode) -> await searchCompleted` | Indents the node at the provided path by replacing it with a group described by `newNode` with the target node as its children. You can await this for when updates settle and relevant searches are completed. **NOTE:** This should rarely be used directly and is mostly an internal utility used by `indent`. You probably want `indent` which has the same signature. |
+| indentInPlace | `async (path, newNode) -> await searchCompleted` | Indents the node at the provided path by mutating it into the node described by `newNode` with the target node as its children. You can await this for when updates settle and relevant searches are completed. This is most useful for indenting the root node of a tree. **NOTE:** This should rarely be used directly and is mostly an internal utility used by `indent`. You probably want `indent` which has the same signature. |
+| move | `async (path, { path, index }) -> await searchCompleted` | Moves the node at the provided path (first argument) to the target location provided in the second parameter. If the target path is not specified, it will default to the current node's group. If the target index isn't provided, it will default to moving to the end of the target group. You can await this for when updates settle and relevant searches are completed. Useful for drag and drop query builder interfaces that let you move nodes around in and between groups. |
+| pauseNested | `async (path) -> await searchCompleted` | Recursively set paused to true for the node at the path and all it's children. You can await this for when updates settle and relevant searches are completed. |
+| unpauseNested | `async (path) -> await searchCompleted` | Recursively set paused to false for the node at the path and all it's children. You can await this for when updates settle and relevant searches are completed. |
+| setPauseNested | `async (path, value) -> await searchCompleted` | Recursively set paused to the passed in `value` for the node at the path and all it's children. You can await this for when updates settle and relevant searches are completed. |
```

Comment:
What's the usecase for this function? If it's flipping we could do so at a higher level and have one less function on our public API.

Reply:
The use case is when you're setting paused based on some flag, where changing methods looks dirtier:

```js
value ? tree.unpauseNested(path) : tree.pauseNested(path)
```
vs
```js
tree.setPausedNested(path, value)
```

In practice it'll probably be a flipPausedNested kind of case:
```js
flipPauseNested = path => tree.setPausedNested(path, tree.isPausedNested(path)
```
vs
```js
flipPauseNested = path => tree.isPausedNested(path) ? tree.unpauseNested(path) : tree.pauseNested(path)
```

We probably don't need both `pause`/`unpause` and `setPaused`. I'm definitely open to killing one to reduce our api surface 👍

Reply:
Although it's worth pointing out that if it's a really common use case, we might want it defined here instead of cropping up all over other repos - that's the reason `isPausedNested` made it here in the first place.

Reply:
It's a common usecase for sure. I just don't think it adds much having it be part of the API for something simple like this (functional utility libraries excluded of course ;) ). Either `isPaused`/`pause`/`unpause` or `isPaused`/`setPaused` looks fine to me.

I know we discarded this option but I think I'm falling in the same black hole you did:
```
F.set({true, false}, tree.pausedNestedLens(path))
F.get(tree.pausedNestedLens(path))
F.flip(tree.pausedNestedLens(path))
```

Reply:
Yup, lenses are very tempting for this... 🤔
More generally, it’s kinda like there’s mutateNested and everyNested, which would combine for a nestedLens... 💭